### DISCUSSION TITLE
Summarize Multi-Footnotes

### DISCUSSION THREAD

**Discussion by frogg** [yuin/goldmark#378] [Ideas]
Hello,
I am using goldmark via Hugo for a blog.
For scientific purposes, we make quite some use of the footnotes `[^1]` feature.
However, sometimes there are multiple footnotes provided for the same word, e.g
`…Research suggests[^1][^2][^3][^4][^5][^6] various associations of smartphone use…`

This turns out to look very ugly:
<img width="306" alt="Screenshot 2023-03-09 at 14 39 04" src="https://user-images.githubusercontent.com/5204169/224043107-3aefdbe0-f1cf-4229-b0fe-f1db6d67aeb2.png">

It would be great if multiple footnotes are separated by comma e.g. `[1,2,3,4,5,6]`, or even better: summarized, e.g. `[1-6]`

**Comment by cmpute** [comment]
Markdown footnotes are usually not intended as academic references. You can generally pack multiple sources into a single footnote, since we are not limited by the academic reference styles where you have to use a consistent reference format.
